### PR TITLE
Update CrazyGamesAdApi.json

### PR DESCRIPTION
**Pull request by Bouh**
New actions:
- Display ad banner
- Display video midgame, and rewarded video ads.
- Display and generate an invite link button
- Display an Happy Time!
- Hide the invite button
- Hide banner(s)

New expression:
- Get the invite link

Tested on the game example https://github.com/GDevelopApp/GDevelop-examples/pull/610, and with the QA tool on CrazyGames' website

### CODE REVIEW

File: extensions/reviewed/CrazyGamesAdApi.json [hunk 1]
```diff
@@ -561,6 +627,291 @@
...
+              "parameterObjects": "",
+              "useStrict": true,
+              "eventsSheetExpanded": true
+            }
+          ]
+        }
+      ],
+      "parameters": [
+        {
+          "description": "Banner's name",
```
Comment (D8H) [inline extensions/reviewed/CrazyGamesAdApi.json:717]: A banner is not a person.
```suggestion
          "description": "Banner name",
```

File: extensions/reviewed/CrazyGamesAdApi.json [hunk 2]
```diff
@@ -561,6 +627,291 @@
...
```
Comment (D8H) [inline extensions/reviewed/CrazyGamesAdApi.json:823]:
```suggestion
          "description": "Banner name",
```

File: extensions/reviewed/CrazyGamesAdApi.json [hunk 3]
```diff
@@ -561,6 +627,291 @@
...
+            ""
+          ],
+          "parameterObjects": "",
+          "useStrict": true,
+          "eventsSheetExpanded": false
+        }
+      ],
+      "parameters": [
+        {
+          "description": "Banner's name",
```
Comment (D8H) [inline extensions/reviewed/CrazyGamesAdApi.json:868]:
```suggestion
          "description": "Banner name",
```

File: extensions/reviewed/CrazyGamesAdApi.json [hunk 4]
```diff
@@ -561,6 +627,291 @@
...
+      "objectGroups": []
+    },
+    {
+      "async": true,
+      "description": "Display a banner that can be called once per 60 seconds.",
+      "fullName": "Display a banner",
+      "functionType": "Action",
+      "group": "Banner",
+      "name": "DisplayBanner",
+      "sentence": "Display a banner: _PARAM1_ at position x:_PARAM3_ and y:_PARAM4_ with size _PARAM2_ ",
```
Comment (D8H) [inline extensions/reviewed/CrazyGamesAdApi.json:731]: to be consistent with other extensions
```suggestion
      "sentence": "Display a banner: _PARAM1_ at position _PARAM3_ ; _PARAM4_ with size _PARAM2_ ",
```

File: extensions/reviewed/CrazyGamesAdApi.json [hunk 5]
```diff
@@ -561,6 +627,291 @@
...
       "objectGroups": []
+    },
+    {
+      "async": true,
+      "description": "Display a responsive banner that can be called once per 60 seconds.",
+      "fullName": "Display a responsive banner",
+      "functionType": "Action",
+      "group": "Banner",
+      "name": "DisplayResponsiveBanner",
+      "private": true,
```
Comment (D8H) [inline extensions/reviewed/CrazyGamesAdApi.json:638]: Why is it private?

File: extensions/reviewed/CrazyGamesAdApi.json [hunk 6]
```diff
@@ -561,6 +627,291 @@
...
+      ],
+      "parameters": [],
+      "objectGroups": []
+    },
+    {
+      "description": "Hide the invite button.",
+      "fullName": "Hide invite button",
+      "functionType": "Action",
+      "group": "Invite",
+      "name": "HideInviteButton",
```
Comment (D8H) [inline extensions/reviewed/CrazyGamesAdApi.json:902]: Is there an action to display the invite button?

Comment (Bouh) [reply extensions/reviewed/CrazyGamesAdApi.json:902]: Yes this action 
image

File: extensions/reviewed/CrazyGamesAdApi.json [hunk 7]
```diff
@@ -561,6 +627,198 @@
...
+        {
+          "description": "Banner name",
+          "name": "BannerName",
+          "type": "string"
+        }
+      ],
+      "objectGroups": []
+    },
+    {
+      "description": "Hide all banners",
```
Comment (D8H) [inline extensions/reviewed/CrazyGamesAdApi.json:783]:
```suggestion
      "description": "Hide all banners.",
```

File: extensions/reviewed/CrazyGamesAdApi.json [hunk 8]
```diff
@@ -561,6 +627,198 @@
...
+        {
+          "description": "Position Y",
+          "name": "PositionY",
+          "type": "expression"
+        }
+      ],
+      "objectGroups": []
+    },
+    {
+      "description": "Hide a banner",
```
Comment (D8H) [inline extensions/reviewed/CrazyGamesAdApi.json:754]:
```suggestion
      "description": "Hide a banner.",
```

**Review by 4ian — APPROVED** [review]

**Pull request merged**